mkosi/__init__.py

```diff
@@ -2899,8 +2955,14 @@ def install_boot_loader(args: CommandLineArguments, root: str, loopdev: Optional
         return
 
     with complete_step("Installing boot loader"):
-        if args.esp_partno and args.distribution != Distribution.clear:
-            run_workspace_command(args, root, ["bootctl", "install"])
+        if args.esp_partno:
+            if args.distribution == Distribution.clear:
+                pass
+            elif args.distribution in (Distribution.centos, Distribution.centos_epel) and \
+                    is_older_than_centos8(args.release):
```

Comment:
Almost, sorry if my previous wording was unclear. I was referring to line 918.
```suggestion
            elif (args.distribution in (Distribution.centos, Distribution.centos_epel) and 
                  is_older_than_centos8(args.release)):
```
From what I see parentheses are generally preferred in Python over line breaks, sine they explicitly group the predicate.

Reply:
Ah, that makes more sense indeed.